include/fi_util.h

```diff
@@ -218,12 +221,14 @@ struct util_cq {
 	struct slist		err_list;
 	fi_cq_read_func		read_entry;
 	int			internal_wait;
-	fi_cq_progress_func	progress;
+	ofi_cq_progress_func	progress;
+	ofi_cq_strerror_func	strerror;
```

Comment:
strerror is already a CQ specific callback as part of the main API.  We shouldn't need this.  The utility providers just need to define their own strerror routines and set it accordingly.  This may mean that the util_cq_strerror routine is not used by the layered providers.